ToDoApp/components/checkBox/CheckBox.js

```diff
@@ -1 +1,26 @@
-//File
+import React, { Component } from 'react';
+import { Image, View, StyleSheet} from 'react-native';
+
+class CheckBox extends Component {
+    render() {
+      return (
+        <View style={{ flex: 1, justifyContent: "flex-end", alignItems: "flex-end", backgroundColor: 'white'}}>
+        {
+            this.props.checked ? <Image source = {require('../../resources/images/iconCheckboxActive.png')} /> : <Image source = {require('../../resources/images/iconCheckboxInactive.png')} />
+        }
+        </View>
+      );
+    }
+}
+module.exports = CheckBox;
+
+const styles = StyleSheet.create({
+    bigBlue: {
+      color: 'blue',
```

Comment:
same about colors

Reply:
I have just changed that, in the next push it will be added.